from .db import *

def save_exception(message):
    save_exception_to_db(message)